web/src/test/manual/web/pr10506/dotty_keys/source/dotty_keys.keyman-touch-layout

```diff
@@ -1,6 +1,6 @@
 {
   "phone": {
-    "font": "DejaVu Dots",
+    "font": "Dotty Keys",
```

Comment:
Contrast:

https://github.com/keymanapp/keyman/blob/d3eb3dd507f46004788f84c0f7c0154bf3e46af9/web/src/test/manual/web/pr10506/header.js#L7-L14

The mismatch here - `"Dotty Keys" != "DejaVu Dots"` - is what the issue and fix are all about.  There are a number of currently-published keyboards with such a mismatch between stub and touch-layout font-family specs.